docs/source/conf.py

```diff
@@ -118,28 +123,62 @@
         .nboutput .prompt {
             display: none;
         }
+
+        .output_area {
+            max-height: 300px;
+            overflow: auto;
+        }
     </style>
 
-    <p>
-        <a style= "background-color:white;color:black;padding:4px 12px;text-decoration:none;display:inline-block;border-radius:8px;box-shadow:0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 3px 10px 0 rgba(0, 0, 0, 0.19)" href="https://colab.research.google.com/github/cleanlab/cleanlab/blob/master/docs/source/{{ docname|e }}" target="_blank">
-            <img src="https://colab.research.google.com/img/colab_favicon_256px.png" alt="Google Colab Logo" style="width:40px;height:40px;vertical-align:middle">   
-            <span style="vertical-align:middle">Run in Google Colab</span>
-        </a>
-    </p>
+    <script type="text/javascript">
+        window.addEventListener('load', () => {
+            const h1_element = document.getElementsByTagName("h1");
+            h1_element[0].insertAdjacentHTML("afterend", `
+            <p>
+                <a style="background-color:white;color:black;padding:4px 12px;text-decoration:none;display:inline-block;border-radius:8px;box-shadow:0 2px 4px 0 rgba(0, 0, 0, 0.2), 0 3px 10px 0 rgba(0, 0, 0, 0.19)" href="https://colab.research.google.com/github/cleanlab/cleanlab/blob/{{ docver|e }}/docs/source/{{ docname|e }}" target="_blank">
```

Comment:
This "run in Google Colab" will always use the `master` branch. This seems incompatible with how we're versioning docs. If we have an old version of Cleanlab with an old tutorial, and then we delete the tutorial in a later version of Cleanlab, the "Run in Colab" will break. And it'll be confusing as we update the tutorials and there's a mismatch between what is rendered (for old versions) and what happens when people click on the Run in Colab button.

Reply:
done!

the code initially does change the URL depending on the version, but after checking it more thoroughly, i noticed that it isn't working as expected. its all fixed now.